### PR TITLE
doc suggestions 1.0

### PR DESCRIPTION
**Pull request by rpanosSFDC**
Months ago when I installed TransmogrifAI, I took some notes on my struggles.  These suggestions may help others.

### CODE REVIEW

File: docs/installation/index.md [hunk 1]
```diff
@@ -1,7 +1,16 @@
 # Installation
 
-* Download and install Java 1.8, then set an environment variable: `export JAVA_HOME=$(/usr/libexec/java_home -v 1.8)`
-* Get Spark 2.3.x: [Download](https://spark.apache.org/downloads.html), unzip it and then set an environment variable: `export SPARK_HOME=<SPARK_FOLDER>`
+* Download and install Java 1.8
+    * To stay sane, some prefer to use JENV to manage different versions of Java: [Recommended Blog](https://medium.com/@danielnenkov/multiple-jdk-versions-on-mac-os-x-with-jenv-5ea5522ddc9b)
+    * On a Mac you might install Java 1.8 with: `brew cask install adoptopenjdk/openjdk/adoptopenjdk8`
```
Comment (crupley) [inline docs/installation/index.md:5]: We've been recommended to use the `zulu8` build of open jdk. Not sure what difference that makes.

**Review by crupley — COMMENTED** [review]
LGTM but would like a second opinion on the java version.

File: docs/installation/index.md [hunk 2]
```diff
@@ -1,7 +1,16 @@
...
+* Determine which Java version you are using (See JENV above):
+  * `$ which java`
+  * `/Users/<user>/.jenv/shims/java`
+* Set the JAVA_HOME environment variable.  Examples:
```
Comment (tovbinm) [inline docs/installation/index.md:9]: please remove extra spaces

File: docs/installation/index.md [hunk 3]
```diff
@@ -1,7 +1,16 @@
...
+* Download and install Java 1.8
+    * To stay sane, some prefer to use JENV to manage different versions of Java: [Recommended Blog](https://medium.com/@danielnenkov/multiple-jdk-versions-on-mac-os-x-with-jenv-5ea5522ddc9b)
+    * On a Mac you might install Java 1.8 with: `brew cask install adoptopenjdk/openjdk/adoptopenjdk8`
+* Determine which Java version you are using (See JENV above):
+  * `$ which java`
+  * `/Users/<user>/.jenv/shims/java`
+* Set the JAVA_HOME environment variable.  Examples: 
+  * `export JAVA_HOME=$(/usr/libexec/java_home -v 1.8)`
+  * `export JAVA_HOME=$(/Users/<user>/.jenv/shims/java -v 1.8)`
+* Get Spark 2.4.x: [Download](https://spark.apache.org/downloads.html), unzip it and then set an environment variable: `export SPARK_HOME=<SPARK_FOLDER>`
```
Comment (tovbinm) [inline docs/installation/index.md:12]: get spark 2.4.5

File: docs/installation/index.md [hunk 4]
```diff
@@ -1,7 +1,16 @@
...
+    * some users also found this useful: [Recommended Blog](https://www.tutorialkart.com/apache-spark/how-to-install-spark-on-mac-os/)
```
Comment (tovbinm) [inline docs/installation/index.md:13]: We should only expose one way of installing software. Too many options can be confusing. I would remove this link.

File: docs/installation/index.md (hunk 3 above)
Comment (rpanosSFDC) [reply docs/installation/index.md:12]: 🙏  I am sure that is key!

File: docs/installation/index.md (hunk 4 above)
Comment (rpanosSFDC) [reply docs/installation/index.md:13]: And I am sure they become out of date easily . . .

File: docs/installation/index.md (hunk 1 above)
Comment (rpanosSFDC) [reply docs/installation/index.md:5]: COOL!  Updated and tested (build step)

**Comment by rpanosSFDC** [comment]
Sorry for the delay and thank you for letting me contribute!

File: docs/installation/index.md [hunk 5]
```diff
@@ -1,7 +1,18 @@
 # Installation
 
-* Download and install Java 1.8, then set an environment variable: `export JAVA_HOME=$(/usr/libexec/java_home -v 1.8)`
-* Get Spark 2.3.x: [Download](https://spark.apache.org/downloads.html), unzip it and then set an environment variable: `export SPARK_HOME=<SPARK_FOLDER>`
+* Download and install Java 1.8
+    * To stay sane, some prefer to use JENV to manage different versions of Java: [Recommended Blog](https://medium.com/@danielnenkov/multiple-jdk-versions-on-mac-os-x-with-jenv-5ea5522ddc9b)
+    * On a Mac you might install Java 1.8 (on Mac) with: `brew cask install homebrew/cask-versions/zulu8`
```
Comment (tovbinm) [inline docs/installation/index.md:5]: @rpanosSFDC this line repeats `Mac` twice

Comment (rpanosSFDC) [reply docs/installation/index.md]: [hyper-embarrassed face]

Thank you.

**Pull request merged**